### PR TITLE
7258 add listener for initial reporter and Market State.

### PR DESCRIPTION
**Pull request by bthaile**
https://app.clubhouse.io/augur/story/7258/make-sure-market-state-is-updated-after-initial-report

Saw that MarketState is now emitted by augur-node having to do with market transition states based on time movement. This PR seemed to be a good place to add it.

### CODE REVIEW

**Comment by sharkcrayon** [comment]
This is what I'm seeing for "0 days left":

screen shot 2018-02-28 at 10 15 51 pm

When there are 0 days left, the white bar should meet the very right edge of the gray area (showing 100% progress), which is why I was mentioning having the text switch sides when it's getting close to that.

**Comment by bthaile** [comment]
ok, moving discussion to https://app.clubhouse.io/augur/story/5838/dispute-view-header

**Comment by justinbarry** [comment]
@bthaile @JohnDanz  added some tests for this file. Please add the relevant tests there.

File: src/modules/app/actions/listen-to-updates.js [hunk 1]
```diff
@@ -37,6 +37,15 @@ export function listenToUpdates(history) {
       }
     })
     augur.events.startAugurNodeEventListeners({
+      InitialReportSubmitted: (err, log) => {
```
Comment (justinbarry) [inline src/modules/app/actions/listen-to-updates.js]:
```
  describe('InitialReportSubmitted', () => {
    let testStore
    beforeEach(() => {
      testStore = mockStore.mockStore({})
    })

    it('should dispatch loadMarketsInfo action', () => {
      const mockAugur = {
        events: {
          startAugurNodeEventListeners: () => {
          },
          stopBlockListeners: () => {
          },
          stopAugurNodeEventListeners: () => {
          },
          stopBlockchainEventListeners: () => {
          },
          startBlockListeners: () => {
          },
          nodes: {
            augur: {
              on: () => {
              },
            },
            ethereum: {
              on: () => {
              },
            },
          },
        },
      }

      ReWireModule.__Rewire__('augur', mockAugur)

      const startAugurNodeEventListenersSpy = sinon.spy(mockAugur.events, 'startAugurNodeEventListeners')

      // Setup listeners
      listenToUpdates(mockHistory)(testStore.dispatch, testStore.getState)

      const listenerMap = startAugurNodeEventListenersSpy.firstCall.args[0]
      listenerMap.InitialReportSubmitted(null, {
        market: '0x12345',
      })

      store.getActions()

    })
  })
```

Comment (bthaile) [reply src/modules/app/actions/listen-to-updates.js]: John added unit tests, I'll add more for the events I'm adding.

**Comment by bthaile** [comment]
Added tests for additional emitter log types.

**Pull request merged**